api/components/v1alpha1/kueue_types.go

```diff
@@ -124,11 +125,22 @@ type KueueManagementSpec struct {
 	ManagementState operatorv1.ManagementState `json:"managementState,omitempty"`
 }
 
+// +kubebuilder:object:generate=true
+type KueueDefaultQueueSpec struct {
```

Comment:
Do you already handle creating resource flavors?

I think you want that if you want Kueue to function out of the box with defaults.

Reply:
not in this PR, it will be done with another PR but likely in the next sprint